### PR TITLE
Fixes transfer totals bug

### PR DESCRIPTION
**Pull request by NateHogan**
Should hopefully fix
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/45498
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/45499

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs [hunk 1]
```diff
@@ -47,10 +47,10 @@ public TransferEvidenceTonnageViewModel Map(TransferEvidenceNotesViewModelMapTra
                 if (source.TransferAllTonnage)
                 {
                     category.TotalReceived = source.Notes.SelectMany(n => n.EvidenceTonnageData)
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs:49]: There should be a unit test to update for this? If not we should create a unit test to catch this bug.

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs [hunk 2]
```diff
@@ -76,11 +76,11 @@ public TransferEvidenceTonnageViewModel Map(TransferEvidenceNotesViewModelMapTra
                         string reusedTonnage = null;
                         if (source.TransferAllTonnage)
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs:77]: Same for this either unit test is currently wrong or we need to create one...

File: src/EA.Weee.Core/AatfEvidence/EvidenceTonnageData.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@ public class EvidenceTonnageData : EvidenceTonnageDataBase
         {
             get 
             {
-                var availble = (Received - TransferredReceived);
+                var availble = (Received - (TransferredReceived.HasValue ? TransferredReceived : 0));
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core/AatfEvidence/EvidenceTonnageData.cs:13]: This should be covered by unit test...

Comment (NateHogan) [reply src/EA.Weee.Core/AatfEvidence/EvidenceTonnageData.cs:13]: Added unit tests

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs (hunk 2 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs:77]: Updated the unit test, I believe should be covered now

**Pull request merged**